src/api/svg.ts

```diff
@@ -76,6 +79,19 @@ interface SVGElementToDrawMap {
   [cmd: string]: (a: SVGElement) => Promise<void>;
 }
 
+const computeDotDistance = (d1: Coordinates, d2: Coordinates) => {
+  return Math.sqrt((d1.x - d2.x) ** 2 + (d1.y - d2.y) ** 2)
+}
+
+const inRange = (v: number, min: number, max: number) => {
```

Comment:
What does it means, and what are those parameters ? What is this goal of this function ? This should normally be one of:

```js
const isInRange = (x, min, max) => min <=x && x<=max
const inRange = (x, min, max) => Math.max(Math.min(x, max), min)
```

Reply:
indeed, I overcomplicated this function